src/main/java/de/teamlapen/vampirism/entity/converted/BiteableEntryManager.java

```diff
@@ -94,6 +97,57 @@ BiteableEntry calculate(@Nonnull CreatureEntity creature) {
         }
     }
 
+    /**
+     * checks if the entity type is blacklisted for blood calculation
+     * does not check if the entity is blacklisted in the server config ({@link #isEntityTypeBlacklisted(EntityType)})
+     *
+     * @param type the entity type
+     * @returns weather the entity type is blacklisted or not
+     */
+    public boolean isEntityTypeBlacklistedByDefault(EntityType<?> type) {
+        if (!VampirismConfig.SERVER.autoCalculateEntityBlood.get()) return true;
```

Comment:
Probably want to move this check to the top of `calculate`. Not sure if there was a reason to have it after the blacklist#contains check beforehand

Reply:
fixed